### PR TITLE
feature: initialized server

### PR DESCRIPTION
**Pull request by doemser**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import React, { useState } from "react";
 import { v4 as uuid } from "uuid";
+
+
```
Comment (marwinburesch) [inline src/App.jsx:4]: This seems like an unrelated change

File: server.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+app.get('/', (request, response) => {
+	response.send('Hello World!')
+})
+
+app.get('/api/todos', (request, response) => {
+	response.json([{id: '1', name: "Apfels kaufen"},{id: '2', name: "Unterwäsche waschen"}])
+})
+
+app.listen(port, () => {
+	console.log(`Example app listening on port ${port}`)
```
Comment (marwinburesch) [inline server.js:15]: I'd change the text here, just to not have the tutorial message

**Review by marwinburesch — COMMENTED** [review]
👍

File: src/App.jsx (hunk 1 above)
Comment (doemser) [reply src/App.jsx:4]: how did that happen?

File: server.js (hunk 2 above)
Comment (doemser) [reply server.js]: Done :)
